lease-kubernetes/src/test/resources/application.conf

```diff
@@ -0,0 +1,3 @@
+akka.coordination.lease.kubernetes {
```

Comment:
by the way, I typically prefer to have test config in the test code itself instead of a "global" application.conf